scripts/check-peer-deps.js

```diff
@@ -68,7 +68,7 @@ async function main() {
   const peerDependenciesWithoutReact = peerDependencies.filter((name) => {
     return !excludedFromProdDependencies.has(name);
   });
-  const extraneousInProdDeps = peerDependencies.filter((name) => {
+  const extraneousInProdDeps = prodDependencies.filter((name) => {
```

Comment:
I should probably modify the `hasError` variable down at Line: 86 to also check for `extraneousInProdDeps.length` ?

Reply:
I think so, yeah? @gribnoysup wrote this so I’d probably defer to him if you want to be sure :)

Reply:
Checking for an error state that we don't expect is covered by `prodDependenciesMatching`:

```js
const prodDependenciesMatching =
  peerDependenciesWithoutReact.length === prodDependencies.length &&
  peerDependenciesWithoutReact.every((name) => {
    return prodDependencies.includes(name);
  });
```

`extraneousInProdDeps` was added to make a more meaningful error message when mismatch happens specifically for dependencies that cloud asked us to keep out of prod deps even when it would make sense otherwise

Reply:
Yea I see it now. Perfect then, thank you :)